utilty.js

```diff
@@ -0,0 +1,39 @@
+const results = document.getElementById('result-span');
+const guesses = document.getElementById('guesses-span');
+const refreshButton = document.getElementById('refresh-button');
+const submitButton = document.getElementById('submit-button');
+
+const input = document.getElementById('input');
+const myNumber = Math.floor((Math.random() * 20)+1)
+let guessesLeft = 4
+
+console.log(myNumber);
+
+
+export function guessingGame(){
+    
+    if (input.valueAsNumber === myNumber){
```

Comment:
We talked about this yesterday but great use of input.valueAsNumber, it was an excellent idea and execution.